app/js/handlebarsQuestionTable.js

```diff
@@ -0,0 +1,102 @@
+let baseUrl = getBaseUrl()
+let questionsTable = {data: []}
+let questionEdited = {
```

Comment:
Make this into a function that returns this object then call it where you need it

Reply:
Done